apps/shell/tash_init.c

```diff
@@ -58,6 +58,11 @@
 #include <tinyara/usb/cdcacm.h>
 #endif
 #include "tash_internal.h"
+#include <sys/ioctl.h>
```

Comment:
Is this necessary regardless of CONFIG_PM?

Reply:
No